src/main/steps/common/components/radios.test.ts

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable @typescript-eslint/ban-types */
+/* eslint-disable jest/expect-expect */
+import { isFieldFilledIn } from '../../../app/form/validation';
+import { CommonContent } from '../common.content';
+import { languageAssertions } from '../test/languageAssertions';
+
+import { Radios } from './radios';
+
+jest.mock('../../../app/form/validation');
+
+const fieldName = 'isFatherStillAlive';
+const EN = 'en';
+const CY = 'cy';
+
+const enContent = {
+  section: 'Section',
+  title: 'Title',
+  continue: 'Save and continue',
+  saveAsDraft: 'Save as draft',
```

Comment:
we normally use these texts from `common.content.ts`

Reply:
They aren't rendered automatically in this test class for some reason, but they do work as expected in the application